osu.Game.Rulesets.Osu/Mods/OsuModHidden.cs

```diff
@@ -7,32 +7,37 @@
 using osu.Framework.Graphics;
 using osu.Game.Rulesets.Mods;
 using osu.Game.Rulesets.Objects.Drawables;
-using osu.Game.Rulesets.Objects.Types;
 using osu.Game.Rulesets.Osu.Objects;
 using osu.Game.Rulesets.Osu.Objects.Drawables;
+using osu.Game.Rulesets.Objects.Types;
+using osu.Game.Configuration;
 
 namespace osu.Game.Rulesets.Osu.Mods
 {
-    public class OsuModHidden : ModHidden, IApplicableToDrawableHitObjects
+    public class OsuModHidden : ModHidden, IApplicableToDrawableHitObjects, IReadFromConfig
     {
         public override string Description => @"Play with no approach circles and fading notes for a slight score advantage.";
         public override double ScoreMultiplier => 1.06;
 
         private const double fade_in_duration_multiplier = 0.4;
         private const double fade_out_duration_multiplier = 0.3;
+        private bool increaseFirstObjectVisibility = true;
+        private IEnumerable<DrawableHitObject> drawables;

```

Comment:
again, should be unnecessary

Reply:
I think we'll want this, but used in a different way. We'll want to bind to the bindable's valuechanged and apply/revert this - suppose the user changes the setting during the lead-in.

Reply:
interesting edge case. may want to handle at a higher level than per-mod.. not sure.